ci/ci.sh

```diff
@@ -3,9 +3,17 @@ set -xe
 
 # Defaults (in case they are not set by the CI)
 F=${F:-production}
-DC=${DC:-dmd1}
 DIST=${DIST:-xenial}
 
+# Binary/package name deduced based on supplied DMD version
+case "$DMD" in
+    1.*    ) DC=dmd1 ;;
+    2.*.s* ) DC=dmd-transitional ;;
+    *      ) DC=dmd ;;
+esac
+
+apt install -y $DC=$DMD-$DIST
```

Comment:
Mmm, good point. I just like/find convenient to be able to use the Dockerfile as is because I do it regularly. Maybe we can have some sort of utility script to define defaults for manual use or something.